libcxx/include/limits

```diff
@@ -21,9 +21,9 @@ class numeric_limits
 {
 public:
     static constexpr bool is_specialized = false;
-    static constexpr T min() noexcept;
-    static constexpr T max() noexcept;
-    static constexpr T lowest() noexcept;
+    [[nodiscard]] static constexpr T min() noexcept;
+    [[nodiscard]] static constexpr T max() noexcept;
+    [[nodiscard]] static constexpr T lowest() noexcept;
```

Comment:
I don't think we should change the synopsis in the comments, because the comments mirror the standard wording, and the standard isn't saying `[[nodiscard]]`. Ditto below.

Reply:
Done, thanks.